### PR TITLE
Improve breadcrumbs across layouts

### PR DESCRIPTION
**Pull request by spencerldixon**
### Trello card

https://trello.com/c/KzENJ1NN/6378-add-breadcrumb-to-subject-pages-and-explore-teaching-adviser-page?filter=member:spencerldixon

### Context

Pages using the subject-style layout don’t have a breadcrumb currently which is not good for accessibility

### Changes proposed in this pull request

- Render breadcrumbs by default in the header component unless `breadcrumbs: false` is set in the front matter
- Amends layouts to include a header component with front matter included
- Overrides breadcrumbs on a few pages that we don't want them on
- Removes an unused shared context from tests

### CODE REVIEW

**Review by martyn-w — APPROVED** [review]
👍  LGTM

File: app/views/content/help-and-support.md [hunk 1]
```diff
@@ -8,6 +8,7 @@ description: |-
 navigation: 40
 layout: "layouts/minimal"
 talk_to_us: false
+breadcrumbs: false
```
Comment (gemmadallmandfe) [inline app/views/content/help-and-support.md:11]: I don't think this is needed - I know it doesn't have a breadcrumb in production but I think it should unless you disagree @jenhadfield-dfe ?

File: docs/content.md [hunk 2]
```diff
@@ -115,6 +116,16 @@ nested:
 
 Knowing which frontmatter values are available for a page is not always obvious, but the best method is to look at a similar page and copy the frontmatter from there (or look at the page examples in this document).
 
+### Breadcrumbs
+
+Breadcrumbs are included by default on most layouts. You can chose to optionally disable breadcrumbs by setting the following in the front matter:
```
Comment (gemmadallmandfe) [inline docs/content.md:121]: @spencerldixon You say `most layouts` here - is it worth calling out which ones don't? Eg sign-up form pages

**Comment by gemmadallmandfe** [comment]
@spencerldixon : Missing end part of the breadcrumb on https://get-into-teaching-app-review-4148.test.teacherservices.cloud/search

image

**Comment by gemmadallmandfe** [comment]
Should we have breadcrumbs on the **start** page of sign-up forms @jenhadfield-dfe 

eg
* https://get-into-teaching-app-review-4148.test.teacherservices.cloud/mailinglist/signup/name
* https://get-into-teaching-app-review-4148.test.teacherservices.cloud/teacher-training-adviser/sign_up/identity
* https://get-into-teaching-app-review-4148.test.teacherservices.cloud/callbacks/book
* https://get-into-teaching-app-review-4148.test.teacherservices.cloud/feedback

plus events form (no GIT events in dev environment at moment so can't share URL)

**Comment by gemmadallmandfe** [comment]
@jenhadfield-dfe Are we happy to put breadcrumbs on landing pages (eg for paid search/social traffic)?

image

File: app/views/content/help-and-support.md (hunk 1 above)
Comment (jenhadfield-dfe) [reply app/views/content/help-and-support.md:11]: yes agree, would work better with a breadcrumb on this page

**Comment by jenhadfield-dfe** [comment]
> Should we have breadcrumbs on the **start** page of sign-up forms @jenhadfield-dfe
> 
> eg
> 
> * https://get-into-teaching-app-review-4148.test.teacherservices.cloud/mailinglist/signup/name
> * https://get-into-teaching-app-review-4148.test.teacherservices.cloud/teacher-training-adviser/sign_up/identity
> * https://get-into-teaching-app-review-4148.test.teacherservices.cloud/callbacks/book
> * https://get-into-teaching-app-review-4148.test.teacherservices.cloud/feedback
> 
> plus events form (no GIT events in dev environment at moment so can't share URL)

No I don't think we need them here, we have a back link when they start the journey and would feel a bit wrong to switch when they click to the next page

**Comment by jenhadfield-dfe** [comment]
> @jenhadfield-dfe Are we happy to put breadcrumbs on landing pages (eg for paid search/social traffic)?
> 
> image

I think we may need to remove them on landing pages - just by the nature of them as every pixel counts and we need users to see things like calls to action without too much scrolling

**Review by gemmadallmandfe — DISMISSED** [review]
Looks good to me

File: docs/content.md [hunk 3]
```diff
@@ -9,6 +9,7 @@ This documentation aims to be a reference for content editors that want to make
 3. [Content Editing Tips/Info](#content-editing-tips-info)
 	* [Headings](#headings)
 	* [Frontmatter](#frontmatter)
+    * [Breadcrumbs](#breadcrumbs)
```
Comment (martyn-w) [inline docs/content.md:12]: could you indent this a couple of spaces to align with the other bullets? or outdent them? 💅

**Review by martyn-w — APPROVED** [review]
👍 LGTM

**Review by gemmadallmandfe — APPROVED** [review]
Looks good to me from a front-end perspective

**Pull request merged**